pkgs/applications/audio/mmlgui/default.nix

```diff
@@ -83,6 +83,8 @@ stdenv.mkDerivation rec {
 
   passthru.updateScript = unstableGitUpdater {
     url = "https://github.com/superctr/mmlgui.git";
+    # a 0.1 tag years ago, some YYMMDD-preview tags a few months afterwards, no more tags since
+    hardcodeZeroVersion = true;
```

Comment:
Tags started out fine with a nice `v1.0`, then they turned into a less-nice `YYMMDD-preview` before stopping altogether.

I would preferably have this be `1.0-unstable-YYYY-MM-DD`, but that would require teaching the updater about tags (patterns?) it should ignore, or telling it outright what version it should use. The former would need more effort to implement that and has the potential of patterns going stale without attentive maintainers, but might be worthwhile. The latter idea in comparison seems like a bad idea - more things the maintainers need to pay attention to, doesn't really follow the spirit of the guideline IMO.